docs/tutorial.rst

```diff
@@ -108,139 +112,278 @@ For reproducibility we can also choose set seed for the simulator using the
+
+Let's go with ``HapmapII_GRCh37``.
+The next command simulates 4 samples of chromosome 1 from each of the four
+populations, and saves the output to a file called ``afr-america-chr1.trees``.
+For the purposes of this tutorial, we'll also specify a random seed using the
+``-s`` option.
+(Note: This took around 8 minutes to run on a laptop.)
+
+.. code-block:: console
+
+    $ stdpopsim homsap -c chr1 -o afr-america-chr1.trees -s 13 -g HapmapII_GRCh37\
+    --model america 4 4 4 4
+
+We should now have a file called ``afr-america-chr1.trees``.
+Our work with ``stdpopsim`` is done; we'll now switch to a Python console and import
+the ``tskit`` package to load and analyse this simulated tree sequence file.
 
 .. code-block:: python
 
-    species = stdpopsim.get_species("homsap")
-    contig = species.get_contig("chr22", length_multiplier=0.1)
-    model = stdpopsim.PiecewiseConstantSize(species.population_size)
-    samples = model.get_samples(10)
-    engine = stdpopsim.get_default_engine()
-    ts = engine.simulate(model, contig, samples)
+    >>> import tskit
+    >>> ts = tskit.load("afr-america-chr1.trees")
+
+Recall that `genetic divergence` is the probability that two randomly sampled
+chromosomes differ at a nucleotide base.
+For a given pair of populations, a pair-specific divergence value is obtained
+by randomly sampling one chromosome from each population.
+These quantities can be estimated directly from our sample using tskit's
+inbuilt :meth:`tskit.TreeSequence.diversity` method.
+
+By looking at
+`the documentation <https://tskit.readthedocs.io/en/latest/python-api.html#tskit.TreeSequence.divergence>`_
+for this method, we can see that we'll need two inputs: ``sample_sets`` and
+``indexes``.
+Let's think about what these inputs are, and how we can obtain them with 
+Python commands.
+In our case, the sample sets correspond to the lists 
+of sample chromosomes (nodes) from each separate population.
+We can obtain the necessary list of lists like this:
+
+.. code-block:: python
 
+    >>> sample_list = []
+    >>> for pop in range(0, ts.num_populations):
+    ...     sample_list.append(ts.samples(pop).tolist())
+    >>> print(sample_list)
+    [[0, 1, 2, 3], [4, 5, 6, 7], [8, 9, 10, 11], [12, 13, 14, 15]]
 
-Here, we simulate 10% of human chromosome 22 under a constant size
-population model, using the current best estimate of the human
-effective population size from the **TODO add crossref to catalog section**
+Note that the samples with node IDs 0 - 3 are from population 0,
+samples with node IDs 4 - 7 are from population 1 and so on. 
 
-.. :ref:`sec_catalog_homo_sapiens_genome`
+The indexes are the pairs of integer indexes corresponding to the populations
+that we wish to compare.
+We can do this quickly with the ``itertools`` module:
 
+.. code-block:: python
 
+    >>> import itertools
+    >>> inds = itertools.combinations_with_replacement(range(0, ts.num_populations), 2)
+    >>> inds = list(inds)
```

Comment:
Alternatively, could just do
```
inds = [(a, b) for a in range(ts.num_populations) for b in ts.num_populations if a <= b]
```
... I'm not sure which is more transparent?

Reply:
I personally like the `itertools` version

Reply:
I think the itertools version is better and more general, as you can use the same function to generate 3-sets or 4-sets (which are needed by some of the other tskit statistics)